### PR TITLE
scripts: sbom: Version 3.20 of SPDX Licenses

### PR DESCRIPTION
**Pull request by doki-nordic**
Newer version of SPDX License List was released.
This commit updates autogenerated license list
in the west ncs-sbom tool.

### CODE REVIEW

File: scripts/west_commands/sbom/data/spdx-licenses.yaml [hunk 1]
```diff
@@ -364,6 +388,8 @@ FSFUL:
   name: "FSF Unlimited License"
 FSFULLR:
   name: "FSF Unlimited License (with License Retention)"
+FSFULLRWD:
+  name: "FSF Unlimited License (With License Retention    and Warranty Disclaimer)"
```
Comment (peknis) [inline scripts/west_commands/sbom/data/spdx-licenses.yaml:392]:
```suggestion
  name: "FSF Unlimited License (With License Retention and Warranty Disclaimer)"
```

**Review by peknis — APPROVED** [review]
Just a minor suggestion to remove a few extra empty spaces.

**Review by maje-emb — APPROVED** [review]

File: scripts/west_commands/sbom/data/spdx-licenses.yaml (hunk 1 above)
Comment (doki-nordic) [reply scripts/west_commands/sbom/data/spdx-licenses.yaml:392]: > `(With License Retention    and Warranty Disclaimer)`

This is how it is in the official source SPDX database. I can look there if they have some easy was to report that, so maybe in future versions they will fix that. In the HTML report, multiple spaces are automatically merged into one, so this is not an issue for us.

Comment (doki-nordic) [reply scripts/west_commands/sbom/data/spdx-licenses.yaml:392]: https://github.com/spdx/license-list-XML/issues/1908

**Pull request closed without merging**